stix2/sources/filesystem.py

```diff
@@ -71,8 +71,8 @@ def stix_dir(self):
         return self.stix_dir
 
     @stix_dir.setter
-    def stix_dir(self, dir):
-        self.stix_dir = dir
+    def stix_dir(self, dir_):
+        self.stix_dir = dir_
```

Comment:
Same argument here. I'd prefer `directory` to `dir_`. I'd also rather call this `data_dir` than `stix_dir`.